packages/select/README.md

```diff
@@ -0,0 +1,131 @@
+            stroke-width="1"
+            fill="none"
+            fill-rule="evenodd"
+          >
+            <path
+              d="M4.67285687,6 L11.3271431,6 C11.9254697,6 12.224633,6.775217 11.8024493,7.22717749 L8.47530616,10.7889853 C8.21248981,11.0703382 7.78751019,11.0703382 7.52748976,10.7889853 L4.19755071,7.22717749 C3.77536701,6.775217 4.07453029,6 4.67285687,6 Z"
+              id="Path"
+              fill="currentColor"
+            ></path>
+          </g>
+        </svg>
+      </div>
+      <div></div
+    ></span>
+  </button>
+</div>
+```
+
+## Select Properties
+
+| Prop           | Type                                                | Description                                                                                                   | Default     |
+| -------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ----------- |
+| `children`     | `node`                                              | `<Option />` and `<OptionGroup />` elements.                                                                  |             |
+| `className`    | `string`                                            | Adds a className to the outermost element.                                                                    |             |
+| `darkMode`     | `boolean`                                           | Determines whether or not the component will appear in dark mode.                                             | `false`     |
+| `size`         | `'xsmall'` \| `'small'` \| `'default'` \| `'large'` | Sets the size of the component's elements.                                                                    | `'default'` |
+| `id`           | `string`                                            | id associated with the Select component.                                                                      |             |
+| `label`        | `string`                                            | Text shown in bold above the input element.                                                                   |             |
+| `description`  | `string`                                            | Text that gives more detail about the requirements for the input.                                             |             |
+| `placeholder`  | `string`                                            | The placeholder text shown in the input element when an option is not selected.                               | `'Select'`  |
+| `disabled`     | `boolean`                                           | Disables the component from being edited.                                                                     | `false`     |
+| `value`        | `string`                                            | Sets the `<Option />` that will appear selected and makes the component a controlled component.               | `''`        |
```

Comment:
Should we consider allowing ReactElements to be passed here as well? A use case I'm thinking about is when we have Icons that would be part of Option value. Let me know what you think cc @DesignerDave

Reply:
Agreed, just because weird use-cases do come up, and it's low effort to support the flexibility.

Reply:
You're right 🤦 I was thinking this was in reference to the display value for an option for some reason.